src/gpd.tf

```diff
@@ -0,0 +1,97 @@
+resource "azurerm_resource_group" "gpd_rg" {
+  name     = format("%s-gpd-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+locals {
+  gpd_cors_configuration = {
+    origins = ["*"]
+    methods = ["*"]
+  }
+}
+
+# Subnet to host the api config
+module "gpd_snet" {
+  count                                          = var.cidr_subnet_gpd != null ? 1 : 0
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v1.0.51"
+  name                                           = format("%s-gpd-snet", local.project)
+  address_prefixes                               = var.cidr_subnet_gpd
+  resource_group_name                            = azurerm_resource_group.rg_vnet.name
+  virtual_network_name                           = module.vnet.name
+  enforce_private_link_endpoint_network_policies = true
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "gpd_app_service" {
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v1.0.93"
```

Comment:
use latest app service modules.
When u create a new app service/function think if u can create a common app service plan to reduce costs

Reply:
done